gradle/publish.gradle

```diff
@@ -23,7 +23,7 @@ artifacts {
     toPublish sourcesJar
 }
 
-if (project.ext.isRelease) {
+if (project.ext.isMakeJavadoc) {
```

Comment:
The `isMakeJavadoc` is only true when the `makeJavadoc` parametr is put inside. This is wrong, as we need to make JavaDoc during release - with the doRelease parameter passed. Let's see the old code - there was also conditional JavaDoc - let's examine the conditions used.

Reply:
if you want to make `doRelease` to imply `makeJavadoc` you can easily add 

```groovy
         isMakeJavadoc = rootProject.hasProperty("makeJavadoc") || rootProject.hasProperty("doRelease")
``` 

The problem was, that `doRelease` depends on nexus upload parameter.